ha/core/event_analyzer/parser/parser.py

```diff
@@ -84,7 +84,7 @@ def parse_event(self, msg: str) -> HealthEvent:
                 EVENT_ATTRIBUTES.RESOURCE_ID : alert[ALERT_ATTRIBUTES.SENSOR_RESPONSE_TYPE][ALERT_ATTRIBUTES.INFO][ALERT_ATTRIBUTES.RESOURCE_ID],
                 EVENT_ATTRIBUTES.SPECIFIC_INFO : alert[ALERT_ATTRIBUTES.SENSOR_RESPONSE_TYPE][ALERT_ATTRIBUTES.SPECIFIC_INFO]
             }
-
+            Log.debug(f"Parsed {event} schema")
```

Comment:
These debug statements are good for production or forgot to remove?

Reply:
yes, i write it for testing but keeping it as useful in debug mode